examples/go/hello-lifecycle/hello.go

```diff
@@ -8,22 +8,23 @@ import (
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
 	"github.com/mattermost/mattermost-plugin-apps/apps/appclient"
+	"github.com/mattermost/mattermost-plugin-apps/utils/httputils"
 )
 
 //go:embed manifest.json
 var manifestData []byte
 
 const (
 	host = "localhost"
-	port = 8080
+	port = 8083
 )
 
 func main() {
 	// Serve its own manifest as HTTP for convenience in dev. mode.
-	http.HandleFunc("/manifest.json", writeJSON(manifestData))
+	http.HandleFunc("/manifest.json", httputils.HandleJSONData(manifestData))
 
 	// Returns the Channel Header and Command bindings for the app.
-	http.HandleFunc("/bindings", writeJSON([]byte("{}")))
+	http.HandleFunc("/bindings", httputils.HandleJSONData([]byte("{}")))
```

Comment:
Should this be returning an empty array instead?

```suggestion
	http.HandleFunc("/bindings", httputils.HandleJSONData([]byte("[]")))
```

Reply:
Isn't the response here a call response? Hence, `{}` is correct.